python/build-requirements.txt

```diff
@@ -12,7 +12,7 @@ mypy
 tensorflow==2.9.3
 numpy==1.21.6
 pandas==1.2.1
-scipy==1.5.4
+scipy==1.7
```

Comment:
@mohazahran Can you fix the minor version too based on what's available internally?

Reply:
ping

Reply:
1.7 is in the pypi

Reply:
I know that, I am asking you to also fix a minor version for the package like `1.7.x` because sometimes someone uploads a newer version to PyPI and then the autobuilds start picking that up instead. It's just an unnecessary maintenance mess.